src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -24,38 +29,108 @@
  */
 public class AddCommandParser implements Parser<AddCommand> {
 
+    public static final Prefix[] RELEVANT_PREFIXES = new Prefix[]{PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL,
```

Comment:
I like how you grouped the prefixes up into relevant, required and optional prefixes! Makes the code a lot neater and readable.